src/popup/components/TransferSend/TransferSendRecipient.vue

```diff
@@ -85,7 +85,8 @@ export default defineComponent({
   setup(props) {
     const { openModal } = useModals();
     const { activeAccount } = useAccounts();
-    const { getTippingUrlStatus } = useAeTippingUrls();
+    const isAe = computed(() => props.protocol === PROTOCOL_AETERNITY);
+    const { getTippingUrlStatus } = useAeTippingUrls({ ensureFetchedOnInit: isAe.value });
```

Comment:
Please group the composables initialization and move the `isAe` computed further down to group it with other computed props.